Makefile

```diff
@@ -77,10 +77,8 @@ e2e-test:
 
 .PHONY: check-tidy
 check-tidy:
-	go mod tidy
-	git diff --exit-code
-	cd tests
-	go mod tidy
+	go mod tidy -v
```

Comment:
Just wondering how come the `-v` was added here? Just preference?

Reply:
I ran into some issues and just copied what we used in flow-go. it probably doesn't matter